lib/classes/TradeOffer.js

```diff
@@ -477,11 +474,13 @@ TradeOffer.prototype.send = function(callback) {
 	}, (err, response, body) => {
```

Comment:
To possibly work around the race condition where its set to 0 while the poll is going on, but the poll reaches the unknown-check before the polldata is further updated.

I have no idea how likely this is to happen, or if it even can happen, but that was my thoughts.